forgerock-core/src/main/java/org/forgerock/android/auth/encrypt/SecretKeyEncryptor.kt

```diff
@@ -61,7 +61,7 @@ class SecretKeyEncryptorConfig {
     var throwWhenEncryptError = true
     var symmetricKeySize = 256
     var invalidatedByBiometricEnrollment = true
-    var strongBoxPreferred = true
+    var strongBoxPreferred = false //Default to false to support more devices
```

Comment:
Should we add some unit test to verify the backward compatibility?

Reply:
Understood. @spetrov, please collaborate with @witrisna to develop a comprehensive manual test plan that includes a variety of devices, specifically focusing on the problematic Samsung models.